src/cli/commands/init.js

```diff
@@ -136,6 +137,9 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
         }
       } else {
         answer = (await reporter.question(question)) || def;
+        if (entry.isShorthand && entry.isShorthand.exec(answer)) {
```

Comment:
Use `entry.isShorthand.test(answer)` instead.

Also not sure why you need to test `entry.isShorthand` since you define it above?